cache/infinispan/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 quarkus.infinispan-client.hosts=localhost:11222
 quarkus.cache.infinispan.expiring-cache.lifespan=3s
 quarkus.cache.infinispan.expiring-cache.max-idle=5s
+quarkus.native.auto-service-loader-registration=true
```

Comment:
My understanding is that this is a workaround QUARKUS-4612 so you should leave a comment as we don't want to keep it

Reply:
Exactly! Added the comment.